resources/create_modulefile.rb

```diff
@@ -0,0 +1,34 @@
+resource_name :create_modulefile
+provides :create_modulefile
+
+property :source_path, String, name_property: true
+property :modulefile_dir, String, required: true
+property :modulefile, String, required: true
+
+default_action :run
+
+action :run do
+
+    # Install env2
+    env2 = "#{node['cfncluster']['scripts_dir']}/env2"
+    remote_file env2 do
+      source node['cfncluster']['env2']['url']
+      mode 0755
+      retries 3
+      retry_delay 5
+      not_if { ::File.exist?(env2) }
+    end
+
+    directory #{new_resource.modulefile_dir}
```

Comment:
isn't this `directory new_resource.modulefile_dir`?

Reply:
changed